### PR TITLE
Added Unit Tests

### PR DESCRIPTION
**Pull request by dehowef**
Unit Tests for Vertexes, Edges, Vertex Arrays, Edge Arrays, Graph IDs, and Paths.

### CODE REVIEW

File: lib/agens.pegjs [hunk 1]
```diff
@@ -1,26 +1,26 @@
...
 {
     var EdgeList = []
     var VertexList = []
     var PathList = []
 
-    var GID = function(oid, id){
+    var GraphId = function(oid, id){
     	this.oid = oid;
-    	this.id = id;
+		this.id = id;
```
Comment (bylee5) [inline lib/agens.pegjs]: What is indentation?

File: test/unit/edgeArrayUnitTest.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
+var assert = require('assert');
+var ag = require('../../lib');
+var agens = require('../../lib/agens.js');
+var g = require('./testFunctions.js')
+
+describe('VertexUnitTest suite', function() {
+
+    var edgeArray = agens.parse('[e1[6.6][7.9,8.3]{"s1":"a"},e2[5.7][2.3,4.8]{"s2":"b"},e3[9.3][3.2,4.4]{"s3":"c"}]', {startRule: 'EdgeArray'});
```
Comment (bylee5) [inline test/unit/edgeArrayUnitTest.js]: Do attributes need quotes?

Comment (dehowef) [reply test/unit/edgeArrayUnitTest.js]: Fixed it, thanks for pointing that out :+1:

File: lib/agens.pegjs (hunk 1 above)
Comment (dehowef) [reply lib/agens.pegjs]: Fixed it, thanks for pointing it out :+1:

File: test/unit/edgeArrayUnitTest.js (hunk 2 above)
Comment (dehowef) [reply test/unit/edgeArrayUnitTest.js]: This is a value returned by the server and should have a valid JSON format, so I think it needs to have double quotes.

File: test/unit/testFunctions.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+function num2GraphId(numstr) {
+        var numpart = numstr.toString().split('.')
+        return new GraphId(numpart[0],numpart[1])
+    }
+
+
+var Vertex = function(label, id, props){
+    	this.label = label;
+		  this.id = new num2GraphId(id);
```
Comment (bylee5) [inline test/unit/testFunctions.js]: What is indentation?

File: test/unit/testFunctions.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    	this.props = props;
+};
+
+var Edge = function(label, id, sid, eid, props){
+    	this.label = label;
+		  this.id = new num2GraphId(id);
```
Comment (bylee5) [inline test/unit/testFunctions.js]: What is indentation?

**Review by bylee5 — APPROVED** [review]

**Pull request merged**